library/OrderCloud.Catalyst/Models/PreWebhookResponse.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace OrderCloud.Catalyst
+{
+    /// <summary>
+    /// Sent to Ordercloud in repsonse to a pre-webhook. 
+    /// </summary>
+    public class PreWebhookResponse
+    {
+        /// <summary>
+        /// Returns a webhook response indicating that OrderCloud.io should proceed with normal processing of the request.
```

Comment:
Can you replace OrderCloud.io with just OrderCloud? I know that got some traction at some point but I think just OrderCloud is sufficient

Reply:
updated